### ISSUE TITLE
nixcloud.TLS default email

### ISSUE THREAD

**Issue by fgaz** [nixcloud/nixcloud-webservices#35]
It would be nice if there was a way to specify a global default email in addition to the per-cert ones

**Comment by qknight** [comment]
so according to the documentation: https://github.com/nixcloud/nixcloud-webservices/blob/master/documentation/nixcloud.TLS.md

we have this syntax:

```
nixcloud.TLS.certs = {
  "example.com-ACME" = {
    domain = "example.com";
    mode = "ACME";
    email = "foo@example.com";
    reload = [ "postifx.service" "myservice.service" ];
  };
  "example.com-selfsigned" = {
    domain = "example.com";
    mode = "selfsigned";
    email = "foo@example.com";
  };
  "example.com-usersupplied" = {
    domain = "example.com";
    mode = {
      tls_certificate="/root/.tls/fullchain.pem";
      tls_certificate_key="/root/.tls/key.pem";
    };
    email = "foo@example.com";
  };
};
```

and you propose this:

```
nixcloud.TLS = {
  email = "foo@example.com";
  certs = {
    "example.com-ACME" = {
      domain = "example.com";
      mode = "ACME";
      reload = [ "postifx.service" "myservice.service" ];
    };
    "example.com-selfsigned" = {
      domain = "example.com";
      mode = "selfsigned";
    };
    "example.com-usersupplied" = {
      domain = "example.com";
      mode = {
        tls_certificate="/root/.tls/fullchain.pem";
        tls_certificate_key="/root/.tls/key.pem";
      };
    };
  };
};
```

if you really want that, we can implement that. wouldn't be much work.

**Comment by qknight** [comment]
more complicated than expected...

**Comment by qknight** [comment]
implemented in 6c61dc1439ca280fbb0e2636f3d0c7a7db4eeaca

**Comment by fgaz** [comment]
Thanks again!
